eis_toolkit/utilities/miscellaneous.py

```diff
@@ -0,0 +1,71 @@
+from numbers import Number
+from typing import Optional, Union
+
+import numpy as np
+import pandas as pd
+from beartype import beartype
+from beartype.typing import Iterable
+
+from eis_toolkit.checks.dataframe import check_columns_valid
+from eis_toolkit.exceptions import InvalidColumnException
+
+
+@beartype
+def replace_values(
+    data: np.ndarray, values_to_replace: Union[Number, Iterable[Number]], replace_value: Number, copy: bool = False
+) -> np.ndarray:
+    """
+    Replace one or many values in a Numpy array with a new value.
+
+    Args:
+        data: Input data as a numpy array.
+        values_to_replace: Values to be replaced with the specified replace value.
+        replace_value: Value that will replace the specified old values.
+        copy: If the output array is a copy of the input data. Defaults to False.
+
+    Returns:
+        Raster data with replaced values.
+    """
+    if copy:
+        out_data = data.copy()
+    else:
+        out_data = data
```

Comment:
`np.where` always returns a new array so the input array is never modified and these lines do therefore nothing. I do not see any reason why someone would want to modify the original input data directly i.e. have a function with side-effects.